File: src/main/Java/boj_step15/boj1929.java
import java.io.*;
import java.math.BigInteger;
import java.util.StringTokenizer;

public class boj1929 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int m = Integer.parseInt(tokenizer.nextToken());
        int n = Integer.parseInt(tokenizer.nextToken());
        for (int i = m; i <= n; i++) if (BigInteger.valueOf(i).isProbablePrime(10)) writer.append(i + "\n");
        writer.close();
    }
}

File: src/main/boj_step15/boj1929.kt

import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (m, n) = r.readLine().split(" ").map { it.toInt() }
    for (i in m .. n) {
        if(i.toBigInteger().isProbablePrime(10)) w.write("$i\n")
    }
    w.close()
}